src/main/java/com/softserve/rms/entities/User.java

```diff
@@ -0,0 +1,50 @@
+package com.softserve.rms.entities;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity(name = "users")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "users")
+@EqualsAndHashCode(
+      exclude = {"resource_templates"})
+@ToString(
+     exclude = {"resource_templates"})
+
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name ="first_name", nullable = false, length = 50)
+    private String firstName;
+
+    @Column(name ="last_name", nullable = false, length = 50)
+    private String lastName;
+
+    @Column(nullable = false,unique = true, length = 50)
+    private String email;
+
+    @Column(nullable = false, unique = true, length = 50)
+    private String phone;
+
+    @Column(nullable = false, length = 255)
+    private String password;
+
+    @Column(nullable = false)
+    private String status;
+//TODO mapping on roles table
+//    @ManyToOne (fetch = FetchType.LAZY)
+//    @JoinColumn(name = "role_id")
+//    private Role role;
+
+    @OneToMany(mappedBy = "user", orphanRemoval = true)
+    private List<ResourceTemplate> resource_templates = new ArrayList<>();
```

Comment:
It's better to use resourceTemplates field name.
Why do you initialize an empty list of resource templates for a user? What should have a user with admin or registrator role in this list? Including the fact that admin should have access to users and registrator to resources only.